h2o-py/test-requirements.txt

```diff
@@ -5,12 +5,13 @@ pykerberos >= 1.1.8, < 2.0.0; sys.platform != 'win32'
 gssapi==1.5.1
 tabulate==0.8.2
 slam==0.6.0
-cython==0.27.3
+cython==0.27.3; python_version < '3.5'
```

Comment:
we don't have cython version spec to Python '3.5'. Can we put cython version spec from line 37 closer to this one?